.travis.yml

```diff
@@ -7,7 +7,9 @@ cache:
 deploy:
   provider: heroku
   api_key:
-    secure: EXxXOqZYQlIgoDdqHvXLDLmUsMoPOOLcr24OAmJemA/+oSiN53ZZlGPQFurwpMn39P6Ngq4uRzK1r+J/m03GEBmp7TdxMaPRjcuSCmuGlcZ/6Ywv9YH+Zb+lovFYpGUEOz1HTnXhO2XeJIdbTLWBWSAsdAvDMyNGW2Kje8cCbp9bE+rTBqodWTVUibUa+ri7e/Rv7GWIoXNl3gXuPTwvidkAv6oueFwgDP7b/8HkSbStTRGA0PTBLHbyaXAOjV5X9t+hz861pHibpXapUR8UmwRN7DvFwkO8+lde55oDJuInQlK/PqIzX3WPYnRLsX+qChxUdLneB3qZMcNqt2M7DrKk5FtHRKS3/QZL8c3oz0ev3y/MzVHxWsHIFXcm3ewu5KyiRkUP/x6rdmGqHCOPDthRmtNbU/mTHTUQAC5R7zS2NHBQgc+iNOa/HyViC7knE+jf+fT7nyP07WDALpFHD701zwm9DxTjUAyHmDAS4Vzbob+IP0H4BLTZxKjUC1vYyRBQlOrsb938i9qjbQKDqHMuo1MYlKtVgBa0lSrKWXDQTFPQgdjkOssijzlfsa7X4V6ntr2K1wNIgQFZptIn8wABOCudJz6Ya3teszDWGxZvPd5jNR3XIeppxq7t1uYq+NmrhBVe1mRlTgqOBks8MZXDJnTx03XGQseNLWU2o0U=
+    secure: B5Qi98tdVPV+qm/sQQix6Pt0zMkHmY5FTEegUIK31j6EKHrKLuCeyOlSV4USGc5x/7Ek6IcmjHBQtYg4kSU0RjXZm1cHgG6ua4ratSQRo2+joIaRNhFvyTcus6yHPXNhydMDeSRqLXK8GqrPhKhwlxP50PMt7dZG+JAAP1SWVJ16prlkdEsf8D0ccvgmmfL9vHtPZQlfPUiopYiCUsGicXeER5sqcTlV/ab0HZCuj2KDpW/Np3FCH4H5iTz4TOa3GwovjJttK+d41j0AfxiJeyn4fxtCBNgngZ6gObfJHtotX5IllzNnRuzLYixFjGtj434mfo1eI6U2kQZKUsSWzNG765psmkhcA3IcO6JfEQn2XWmFyJuJD4RwbfaD/sMsMLl9WBjKv7DTJPKcw51AStMYC/J66Gig9td/LgLeiModZ0nypj010tte2L8Vm7EjC6yZcsuSX237x2MXLOovItPNsKyQzZ4MOforPuZzs7JxTirbyhuuwvlG1Ae8V73VTfFVOwJfNlRUPVo9goN2jCK17fye4nrJhcLaFwpiSHjTq5L2J5V0j/tKgtzv1LMHYT7v5QHpg78hjuTzno27owyDb8yUpwqQCkW31mlDcCAeQsamKmyEGNi9p22+cIdmoRfrXGeTJlzjfEIFtTVwtu5kzz1dQSDdzZ15WVvELf0=
   app: obscure-waters-10676
   strategy: git
+  on:
```

Comment:
i know you did this to get a deployment out for the meeting tonight, but it can't stay. i'll continue working on this after the meeting